packages/design-system/src/components/dialog/stories/index.js

```diff
@@ -70,22 +80,20 @@ export const _default = () => {
           setToggleDialog(!toggleDialog);
```

Comment:
This wasn't you but seeing an error in the console when opening and closing the modal. I can make a ticket to fix that, but just adding here for me as a reminder
![image](https://user-images.githubusercontent.com/22185279/151241261-ead0e4f5-3e57-44b1-9932-05192e111231.png)

Reply:
new ticket for this is a good idea